app/src/main/java/com/globant/pokemontcgapp/activity/PokemonCardListActivity.kt

```diff
@@ -23,36 +25,43 @@ class PokemonCardListActivity : AppCompatActivity() {
 
     private val pokemonCardListViewModel by viewModel<PokemonCardListViewModel>()
     private lateinit var binding: ActivityPokemonCardListBinding
+    private lateinit var group: String
+    private lateinit var groupSelected: String
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityPokemonCardListBinding.inflate(layoutInflater)
         setContentView(binding.root)
 
-        val selection = intent.getStringExtra(SELECTION)
-        val selectionColor = intent.getIntExtra(SELECTION_COLOR, DEFAULT_VALUE)
-        initUi(selection, selectionColor)
+        group = intent.getStringExtra(POKEMON_GROUP)
+        groupSelected = intent.getStringExtra(SELECTION)
```

Comment:
can you give group vars a more descriptive name?

Reply:
Refactored for a more descriptive name, thanks for the help